src/App.js

```diff
@@ -1,13 +1,83 @@
 import React from 'react';
 
+import TodoForm from './components/TodoComponents/TodoForm';
+import TodoList from './components/TodoComponents/TodoList';
+
+// export const data = [
+//   {
+//     task: 'Organize Garage',
+//     id: 1528817077286,
+//     completed: false
+//   },
+//   {
+//     task: 'Bake Cookies',
+//     id: 1528817084358,
+//     completed: false
+//   }
+// ];
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor () {
+    super();
+    this.state = {
+      list: [
+        // {
+        //   task: 'Organize Garage',
+        //   id: 1528817077286,
+        //   completed: false
+        // },
+        // {
+        //   task: 'Bake Cookies',
+        //   id: 1528817084358,
+        //   completed: false
+        // }
+      ],
+      task: "",
+      id: "",
+      completed: ""
+    };
+  }
+  addToList = e => {
+    // console.log("its working");
+    e.preventDefault();
+    const newTodo = {
+      task: this.state.task,
+      id: this.state.id,
+      completed: this.state.completed
+    }
```

Comment:
This is why you don't need to store those in state, you should be assigning the `id` a random number value. and `completed` should be set to `false` since no task you are adding will already be complete.